.github/workflows/test-build-deploy.yml

```diff
@@ -32,6 +32,24 @@ jobs:
         run: |
           mkdir -p /go/src/github.com/grafana/mimir
           ln -s $GITHUB_WORKSPACE/* /go/src/github.com/grafana/mimir
+      - name: Get golangci-lint cache path
+        id: golangcilintcache
+        run: |
+          echo "path=$(golangci-lint cache status | grep 'Dir: ' | cut -d ' ' -f2)" >> "$GITHUB_OUTPUT"
```

Comment:
Can we also cache $GOMODCACHE? I see the [Go example](https://github.com/actions/cache/blob/main/examples.md#go---modules) also does this.